tpcc/new_order.go

```diff
@@ -245,12 +248,17 @@ func (n newOrder) run(db *sql.DB, wID int) (interface{}, error) {
 				item := &d.items[i]
 
 				if !rows.Next() {
+					fmt.Printf(`%d: missing stock row: retry=%d %s
```

Comment:
I'm having a hard time reasoning about this. I think it would be easier if we made the following changes:
- got rid of the `i` index variable
- determine `j` immediately after the `rows.Scan` call
- move `item := &d.items[i]` down and change it to `item := &d.items[j]`